### PR TITLE
Lab2.1

### PR DESCRIPTION
**Pull request by YuriFrolov**

### CODE REVIEW

**Comment by YuriFrolov** [comment]
Removed list

File: CSharp_Net_module1_2_1_lab/CSharp_Net_module1_2_1_lab/LibraryUser.cs [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            }
+            
+        }
+
+        public int BooksCount()
+        {
+            return _booksCount;
+        }
+
+        public string BookInfo(int index)
```
Comment (Landay7) [inline CSharp_Net_module1_2_1_lab/CSharp_Net_module1_2_1_lab/LibraryUser.cs:104]: Don't agree!!

File: CSharp_Net_module1_2_1_lab/CSharp_Net_module1_2_1_lab/App.config [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (Landay7) [inline CSharp_Net_module1_2_1_lab/CSharp_Net_module1_2_1_lab/App.config:1]: wrong

Comment (Landay7) [reply CSharp_Net_module1_2_1_lab/CSharp_Net_module1_2_1_lab/App.config:1]: not wrong

**Review by Landay7 — APPROVED** [review]

**Pull request merged**